scripts/run_all_tests.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+set -ex
+
+shout "Setting up"
+
+mkdir bin
+GOBIN="`pwd`/bin"
+KUBECONFIG="`pwd`/config"
+SCRIPT_IDENTITY=${SCRIPT_IDENTITY:-"def-id"}
+export SKIP_USER_LOGIN_TESTS="true"
+export GINKGO_TEST_ARGS="--noColor"
+
+shout "Getting oc binary"
+if [[ $BASE_OS == "linux"  ]]; then
+    set +x
+    curl --connect-timeout 150 --max-time 2048 -k ${OCP4X_DOWNLOAD_URL}/${ARCH}/${BASE_OS}/oc.tar -o ./oc.tar
```

Comment:
Where are we setting the link to oc binary?

Reply:
This is done on jenkins side using `--env` parameter of `ci-firewall work` which is run there. So as an example, this could end up looking like
`example.com/ppcle/linux/oc.tar`